include/DCMTrajectoryGeneratorHelper.h

```diff
@@ -106,6 +106,7 @@ class DCMTrajectoryGeneratorHelper
 
     double m_dT; /**< Planner period. */
     double m_omega; /**< Time constant of the 3D-LIPM. */
+    double m_alpha; /**< Time constant of the 3D-LIPM. */
```

Comment:
Please, make sure that this value is initialized somewhere.

Reply:
Also, fix the comment (copy/paste error)

Reply:
Exactly I was doing that! Done!

Reply:
Don't forget the initialization :wink:

Reply:
Yes, I'm doing that! :wink:

Reply:
Done!